openslides_backend/models/motion_comment.py

```diff
@@ -0,0 +1,30 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionComment(Model):
+    """
+    Model for motion comments-
+
+    There are the following reverse relation fields:
+        TODO
```

Comment:
There are none, so you can just delete this TODO.

Reply:
Removed.